index.js

```diff
@@ -121,12 +123,17 @@ function* run() {
     yield rs.start();
   }
 
-  const hosts = ports.map(port => `localhost:${port}`);
+  const hosts = ports.map(port => isWin ? `${hostname}:${port}` : `localhost:${port}`);
   console.log(chalk.green(`Started replica set on "mongodb://${hosts.join(',')}?replicaSet=rs"`));
 
   if (commander.shell) {
     console.log(chalk.blue(`Running mongo shell: ${mongo}`));
-    spawn(mongo, ['--quiet'], { stdio: 'inherit' });
+    const shellDefaultHost = (hosts[0].split(':'))[0];
+    const shellDefaultPort = (hosts[0].split(':'))[1];
+    spawn(mongo, 
+      isWin ? ['--quiet','--port', shellDefaultPort, '--host', shellDefaultHost] : ['--quiet'], 
```

Comment:
Is `--port` necessary here? For now, it'll always be the default `27017`

Reply:
Hi @vkarpov15 , it was after hours of wondering that I ended up learning that every time I ran `run-rs --shell` without specifying the port and host, the shell was connecting to some non-replica set instance of mongodb on port 127.0.0.1:27017! But with `run-rs --port 27017 --host host_name` the shell was able to connect to the `primary node` of the replica set, and because I was tired I just settled to always specify the port and host. lol. If tested and works well on other Windows machines without `--port`, there is no need to keep it there!
Thanks. :)